spec/metrics-spec.js

```diff
@@ -144,6 +144,7 @@ describe("Metrics", async () => {
     describe("when the user is chosen to send commands", async () => {
       beforeEach(async () => {
         localStorage.setItem('metrics.userId', 'd')
+        spyOn(Reporter, 'shouldReportPanesAndCommands').andReturn(true)
```

Comment:
I believe this test would also fail 5 percent of the time previously because we didn't override the random decision.